style.css

```diff
@@ -1 +1,215 @@
 /*  Add your styling here */
+body {
+    display: grid;
+    grid-area: ham;
+    padding-top: 10px;
+    margin-left: 15px;
+    height: 20px;
+}
+
+header > p:nth-child(3) {
+  display: grid;
+  grid-area: par;
+  text-align: end;
+  margin-top: -25px;
+  padding-right: 22px;
+  font-size: 1.1rem;
+}
+header > p:nth-child(4) {
+  display: grid;
+  grid-area: par1;
+  text-align: end;
+  margin-top: -10px;
+  padding-right: 15px;
+  padding-right: 22px;
+  font-size: 1.1rem;
+}
+.logo-img{
+    width:5rem;
+    height: 7rem ; 
+}
+nav{
+  grid-template-areas: inherit;
+  justify-self:end;
+}
+/* welcome section's styles */
+.wlc > p:nth-child(1) {
+  font-size: 3.5rem;
+  margin: 0;
+  padding-left: 0px;
+}
+.wlc > p:nth-child(2) {
+  padding-left: 0px;
+  width: 100%;
+  text-align: justify;
+  margin-bottom: 30px;
+}
+/* intro section style */
+.intro {
+  display: grid;
+  grid-area: intro;
+}
+.wlc {
+  margin-top: 40px;
+}
+.big-img {
+  width: 100%;
+  margin-bottom: 30px;
+}
+/* products style */
+.cake1-img img {
```

Comment:
cake{N}-img styles are identical - do you really need to define multiple of them? Can we simplify this and use the same style for all images?